Header header
Header image_header
BoundingBoxInstance[] bounding_boxes